### PR TITLE
Make cron job parameter configurable

### PR DESCRIPTION
**Pull request by mmalfertheiner**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 postgresql_backup:
   enabled: false
   directory: ''
+  cron: '30 03 * * *'
```
Comment (mmalfertheiner) [inline defaults/main.yml]: @alexlanz 
do I need the quotes here?

File: tasks/main.yml [hunk 2]
```diff
@@ -70,8 +70,8 @@
   when: postgresql_backup.enabled
 
 - name: Setup PostgreSQL backup script
-  ansible.builtin.copy:
-    src: files/postgres
+  ansible.builtin.template:
+    src: templates/postgres.j2
     dest: /etc/cron.d/postgres
```
Comment (mmalfertheiner) [inline tasks/main.yml]: @alexlanz should we give the cron job a more explicit name? postgres-backup?

Comment (alexlanz) [reply tasks/main.yml]: Great idea

File: defaults/main.yml (hunk 1 above)
Comment (alexlanz) [reply defaults/main.yml]:
```suggestion
  time: '30 03 * * *'
```

You don't need the quotes, but is also no problem if you leave them.

What do you think about giving the property a more specific name?

Comment (mmalfertheiner) [reply defaults/main.yml]: I think `cron` is actually more explicit then `time`. 

For example in Spring boot you can schedule jobs and they also name it cron:

`@Scheduled(cron = "30 03 * * *")`

Comment (alexlanz) [reply defaults/main.yml]: Ok, works for me

**Review by alexlanz — APPROVED** [review]

**Pull request merged**